sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/json/JSONRelation.scala

```diff
@@ -79,12 +76,16 @@ private[sql] class JSONRelation(
 
   override val needConversion: Boolean = false
 
-  private def createBaseRdd(inputPaths: Array[FileStatus]): RDD[String] = {
+  private def createBaseRdd(
+      inputPaths: Array[FileStatus],
```

Comment:
Sorry for late reply. Yes, look not good and need not to be handed over. I'll change "paths = sources" to "paths = cachedSourcePaths()"